turbinia/workers/containerd.py

```diff
@@ -142,20 +189,59 @@ def run(self, evidence, result):
           f'Found {len(container_ids)} containers: {", ".join(container_ids)}')
 
       # 2. Add containers as evidences
+      new_evidence = []
       for container in containers:
         namespace = container.get('Namespace')
         container_id = container.get('ID')
+        if container.get('Labels'):
+          pod_name = container.get('Labels').get(
+              POD_NAME_LABEL, 'UnknownPodName')
+        else:
+          pod_name = 'UnknownPodName'
         container_type = container.get('ContainerType') or None
+        image = container.get('Image')
```

Comment:
Could this ever be None? if so, calls to split() might fail but maybe this is never the case.

Reply:
Not sure if it would happen in a real world scenario, but this did come up in the tests so I added a default for this as well.

Reply:
Never mind, i see now there is an if statement here. :)